geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapred/GeoMesaOutputFormat.scala

```diff
@@ -0,0 +1,116 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.jobs.mapred
+
+import java.io.IOException
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.hadoop.fs.FileSystem
+import org.apache.hadoop.io.Text
+import org.apache.hadoop.mapred._
+import org.apache.hadoop.util.Progressable
+import org.geotools.data.{DataStoreFinder, DataUtilities}
+import org.geotools.filter.identity.FeatureIdImpl
+import org.locationtech.geomesa.index.api.{GeoMesaFeatureIndex, WrappedFeature}
+import org.locationtech.geomesa.index.geotools.{GeoMesaDataStore, GeoMesaFeatureWriter}
+import org.locationtech.geomesa.jobs.GeoMesaConfigurator
+import org.locationtech.geomesa.utils.index.IndexMode
+import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
+
+import scala.collection.JavaConversions._
+
+object GeoMesaOutputFormat {
+
+  /**
+   * Configure the data store you will be writing to.
+   */
+  def configureDataStore(job: JobConf, dsParams: Map[String, String]): Unit = {
+    val ds = DataStoreFinder.getDataStore(dsParams).asInstanceOf[GeoMesaDataStore[_, _, _, _]]
+    assert(ds != null, "Invalid data store parameters")
+    ds.dispose()
+
+    // set the datastore parameters so we can access them later
+    GeoMesaConfigurator.setDataStoreOutParams(job, dsParams)
+    GeoMesaConfigurator.setSerialization(job)
+  }
+}
+
+/**
+ * Output format that turns simple features into mutations and delegates to AccumuloOutputFormat
```

Comment:
maybe change comment to not specifically call out just accumulo ... also in mapreduce package